great_expectations_cloud/agent/agent.py

```diff
@@ -581,3 +584,15 @@ def _update_headers_agent_patch(
         # TODO: this is relying on a private implementation detail
         # use a public API once it is available
         http._update_headers = _update_headers_agent_patch
+
+    def _raise_gx_cloud_err_on_http_error(self, response: requests.Response) -> None:
```

Comment:
```suggestion
    @classmethod
    def _raise_gx_cloud_err_on_http_error(response: requests.Response) -> None:
```
This could be a class or static method.